upspeak/index.html

```diff
@@ -186,7 +186,7 @@ <h1 class="display-3 text-center mb-5">FAQ</h1>
                                         data-parent="#accordionExampleTwo"
                                 >
                                     <div class="card-body opacity-8 text-justify">
-                                        Please email info@sefglobal.com with your CV and a
+                                        Please email <a href="info@sefglobal.com">info@sefglobal.com</a> with your CV and a
```

Comment:
```suggestion
                                        Please email <a href="mailto:info@sefglobal.com">info@sefglobal.com</a> with your CV and a
```

Hi @Yoshitha-SACK, we have to append the `mailto:` part to the link. Otherwise, it will redirect to a 404 error page.

Reply:
Done